web.tf

```diff
@@ -9,7 +9,7 @@ resource "aws_security_group" "acme_web" {
   tags = {
     Name = "acme_web"
   }
-
```

Comment:
**Accurics remediation**
 
**[HIGH]: ssh port open to internet**
 
**Vulnerability**
ssh port open to internet
 
**Remediation**
Change cidr_block to non 0.0.0.0/0

source: accurics-app